HW_7/Task_2.py

```diff
@@ -0,0 +1,14 @@
+# Задача 36:
+# Напишите функцию print_operation_table(operation, num_rows=6, num_columns=6),
+# которая принимает в качестве аргумента функцию, вычисляющую элемент по номеру строки и столбца.
+# Аргументы num_rows и num_columns указывают число строк и столбцов таблицы, которые должны быть распечатаны.
+# Нумерация строк и столбцов идет с единицы (подумайте, почему не с нуля).
+# Примечание: бинарной операцией называется любая операция, у которой ровно два аргумента, как, например,
+# у операции умножения.
+
+def print_operation_table(operation, num_rows=6, num_columns=6):
+    a = [[operation(i, j) for j in range(1, num_columns+1)]
+for i in range(1, num_rows+1)]
+    for i in a:
+        print(*[f"{x:>3}" for x in i])
```

Comment:
Такое расположение строк сбивает с толку
```suggestion
def print_operation_table(operation, num_rows=6, num_columns=6):
    a = [[operation(i, j) for j in range(1, num_columns + 1)] for i in range(1, num_rows + 1)]
    for i in a:
        print(*[f"{x:>3}" for x in i])
```
так будет лучше

Reply:
Кстати, у вас конструкция с `for` используется 4 раза, а можете только 1 раз ее использовать?